packages/account/src/providers/message.ts

```diff
@@ -41,7 +44,6 @@ export type BlockHeader = {
   prevRoot: string;
   time: string;
   applicationHash: string;
-  messageReceiptRoot: string;
```

Comment:
This field was renamed to `message_outbox_root`, but still is available=) Also, we have 3 more fields:
- `event_inbox_root`
- `consensus_parameters_version`
- `state_transition_bytecode_version`

Reply:
@xgreenx I've missed that.

Thanks a lot 🙏